pkg/registry/ociregistry.go

```diff
@@ -0,0 +1,164 @@
+package registry
```

Comment:
maybe we can name this file `client.go`?

Reply:
Done.